### PR TITLE
Adding the names.

### PR DESCRIPTION
**Pull request by foresterre**
I hereby commit,
This pull request with git.

Praise the sun and @newnottakenname,
Testing is Jean's bane.

### CODE REVIEW

**Comment by jadeleeuw** [comment]
LGTM

**Review by clanghout — CHANGES_REQUESTED** [review]
Needs more names

**Comment by jadeleeuw** [comment]
Which names would you like to see added?

**Comment by jadeleeuw** [comment]
:+1:

**Comment by clanghout** [comment]
:joy: :100: :b: :ok_hand:

File: liam/index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@ <h1>HAPPY BIRTHDAY LIAM</h1>
     <div id="ticket" class="page">
         <p>We hereby offer you</p>
         <h1>One free ticket to Walibi</h1>
-        <p>From Tim van der Lippe, Thomas Smith and Martijn Steenbergen</p>
+        <p>From Tim van der Lippe, Thomas Smith, Martijn Steenbergen, Jean de Leeuw, Martijn Gribnau, Thomas Overklift, Chak Shun Yu and &quot;uhh ik&quot;.</p>
```
Comment (TimvdLippe) [inline liam/index.html]: Chris be like:

Comment (foresterre) [reply liam/index.html]: https://imgur.com/a/hfaYIy2

**Review by mjwsteenbergen — APPROVED** [review]
Confirms to community guidelines and standards and adds names. Good new feature

**Comment by clanghout** [comment]
CLA (uhh ik) does not approve

**Comment by jadeleeuw** [comment]
> CLA(uhh ik) does not approve

Changes requested with regards to this attitude

**Review by jadeleeuw — CHANGES_REQUESTED** [review]
Who is this "Chris Langhout" fellow? Never heard of him/her/it.

**Comment by mjwsteenbergen** [comment]
As a maintainer, I will make the executive decision to rename uhh ik to Chris again

**Pull request merged**